### DISCUSSION TITLE
mspider/user-guide/traffic-monitor/conn-topo/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1304] [Announcements]
# mspider/user-guide/traffic-monitor/conn-topo/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/user-guide/traffic-monitor/conn-topo/

**Comment by liyuerich** [comment]
这个图现在是不是变了？ 健康状态能和图片上具体服务对应一下吗？

**Reply by liyuerich** [reply]
图片没有问题，请忽略

**Reply by nicole-lihui** [reply]
应该添加工作负载视图
